### PR TITLE
console log api

### PR DESCRIPTION
**Pull request by TempoLab**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
 function App() {
   const [state, setState] = useState('idle');
+  const [userApiValue, setUserApiValue] = useState([])
+
+  const apiHandler = async (userApiValue) => {
+    const search = userApiValue
+    try {
+        setState('loading')
+        const response = await ky.get(`https://api.torn.com/user/?selections=revives&key=${search}`).json()
+        console.log(response)
```
Comment (chopfitzroy) [inline src/App.jsx]: NIT: It's usually bad practice to keep `console.log` in a codebase outside of debugging purposes.

Unless you have a strong reason to keep this I would recommend removing it.

File: src/App.jsx [hunk 2]
```diff
@@ -16,16 +34,16 @@ function App() {
...
+        <input type="text" value={userApiValue} onChange={handleSearchChange} id="userApi" name="userApi" />
       </div>
 
-      <div>
+      {/* <div>
         <label htmlFor="factionFilter">Faction filter:</label>
-        <input type="text" id="factionFilter" name="factionFilter" />
-      </div>
+        <input type="text" value={factionSearchValue} id="factionFilter" name="factionFilter" />
+      </div> */}
```
Comment (chopfitzroy) [inline src/App.jsx]: QUESTION: Why is this commented out? If it is not needed I recommend removing.

File: src/App.jsx [hunk 3]
```diff
@@ -16,16 +34,16 @@ function App() {
...
 
       <div>
-        <button>Get Data!</button>
+        <button onClick={getData}>Get Data!</button>
```
Comment (chopfitzroy) [inline src/App.jsx]: PERF: I recommend adding a `disabled={state === 'loading'}` attribute to this to prevent the user making multiple requests.

**Review by chopfitzroy — CHANGES_REQUESTED** [review]
Looking good, a few minor things.

File: src/App.jsx (hunk 1 above)
Comment (TempoLab) [reply src/App.jsx]: Gone.

File: src/App.jsx (hunk 3 above)
Comment (TempoLab) [reply src/App.jsx]: Added.

File: src/App.jsx (hunk 2 above)
Comment (TempoLab) [reply src/App.jsx]: Fixed.

**Comment by TempoLab** [comment]
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣀⣠⣤⣤⣤⣄⣀⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⣠⡶⠒⠒⠶⣄⣠⡴⠚⠉⠁⠀⠀⠀⠀⠀⠉⠙⠳⢦⡀⠀⠀⠀⠀⠀⠀
⢠⡏⠀⠀⠀⠀⠘⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠙⢧⡀⠀⠀⠀⠀
⢸⡄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠋⢱⠀⠀⢠⠉⢡⠀⠀⠀⠀⠀⠻⡄⠀⠀⠀
⠀⣧⠀⠀⠀⠀⠀⠀⠀⠀⢸⣧⣾⠄⠀⢸⣦⣾⠀⠀⠀⠀⠀⠀⢻⡄⠀⠀
⠀⠘⢧⡀⠀⠀⠀⠀⠀⠀⠈⣿⣿⠀⠀⠸⣿⡿⠀⠀⠀⠀⠀⠀⠈⠳⣄⠀
⠀⠀⠀⡇⠀⠀⠀⠀⠀⠀⠀⠈⠁⡴⠶⡆⠉⠁⠀⠀⠀⠀⠀⠀⠀⠀⠹⡄
⠀⠀⠀⢷⠀⠀⠀⠀⠀⠀⠀⠀⠀⠐⠒⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣷
⠀⠀⠀⠸⡆⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣠⠇
⠀⠀⠀⣀⡿⣆⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⡽⣿⡛⠁⠀
⠀⣠⢾⣭⠀⠈⠳⣄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⡠⠊⠀⢠⣝⣷⡀
⢠⡏⠘⠋⠀⠀⠀⠈⠑⠦⣄⣀⠀⠀⠀⠀⠀⣀⡠⠔⠋⠀⠀⠀⠈⠛⠃⢻
⠈⠷⣤⣀⣀⣀⣀⣀⣀⣀⣀⣤⡽⠟⠛⠿⣭⣄⣀⣀⣀⣀⣀⣀⣀⣀⣤⠞
⠀⠀⠀⠀⠉⠉⠉⠉⠉⠉⠁⠀⠀⠀⠀⠀⠀⠀⠈⠉⠉⠉⠉⠉⠉⠀⠀⠀

**Review by chopfitzroy — APPROVED** [review]

**Pull request merged**